objects/obj_controller/Create_0.gml

```diff
@@ -529,7 +529,7 @@ exit_all=0;
 // 
 diplomacy=0;
 trading=0;
-trading_artifact=0;
+trading_artifat=0;
```

Comment:
this one was fine

Reply:
yeah sorted good catch not sure how that happened